nml/actions/action2var_variables.py

```diff
@@ -562,8 +562,7 @@ def industry_town_count(name, args, pos, info):
     return (args[0], extra_params)
 
 def industry_cargotype(name, args, pos, info):
-    from nml.expression.functioncall import builtin_cargotype
-    return (builtin_cargotype(name, args, pos), [])
+    return (expression.functioncall.builtin_resolve_typelabel(name, args, pos), [])
```

Comment:
builtin_resolve_typelabel would not need that "default to cargo" trick, if there was a 4th optional paramater for the function name, like
`
builtin_resolve_typelabel("cargo", args, pos, username=name)
`

Reply:
> Not a huge fan of the "random" distinction between `@builtin` and `@builtin_names`, and with the naming of the functions. Could they be made more consistent?
> 
> I'd probably go with always having a "builtin_" prefix to the function name and always using `@builtin_names`

I changed `@builtin` to require and strip off the `builtin_` prefix, which removes most of the inconsistency.

Possibly I'm overthinking this and should just use `builtins("name")` for everything, but this way reduces the chance of copy-paste errors if people add new functions.

Reply:
I really don't like this idea. The existing `name` parameter already has that meaning; passing in a fake value to manipulate the behaviour and then adding a new parameter for the real value seems like a far bigger hack. The default is going to exist somewhere, why not here?

(I updated the comment a bit)